src/App.js

```diff
@@ -150,6 +150,7 @@ class App extends Component {
             minimumChange={60}
             maximumLength={3600}
             minimumLength={60}
+            timerState={this.state.timerState}
```

Comment:
instead of timerstate you could have passed if the buttons should be enabled or not. We are passing something to these buttons which it does not need to know.